src/ILCompiler.MetadataTransform/tests/SimpleTests.cs

```diff
@@ -28,13 +29,14 @@ public SimpleTests()
         public void TestAllTypes()
         {
             var policy = new SingleFileMetadataPolicy();
+            policy.Init();
             var transformResult = MetadataTransform.Run(policy, new[] { _systemModule });
 
-            Assert.Equal(1, transformResult.Scopes.Count());
+            Assert.Equal(2, transformResult.Scopes.Count());
             
             Assert.Equal(
                 _systemModule.GetAllTypes().Count(x => !policy.IsBlocked(x)),
-                transformResult.Scopes.Single().GetAllTypes().Count());
+                transformResult.Scopes.First().GetAllTypes().Count() + transformResult.Scopes.Last().GetAllTypes().Count());
```

Comment:
With these changes this is no longer a simple unit test, but it relies on a bunch of other functionality working (explicit scopes) to pass.

I would still prefer you put the ExplicitScope metadata into a separate assembly to keep the unit test being a unit test that tests individual features. Also, the new tests might be better off in a separate file because this no longer feels like a simple test.